loads_test/spec_circular_test.go

```diff
@@ -1,20 +0,0 @@
-package loads_test
-
-import (
-	"testing"
-
-	"github.com/go-openapi/loads"
-	"github.com/go-openapi/strfmt"
-	"github.com/go-openapi/validate"
-	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/require"
-)
-
-func TestPathLoaderIssue(t *testing.T) {
-	swaggerFile := "fixtures/json/resources/pathLoaderIssue.json"
```

Comment:
Why is this removed?

Reply:
@youyuanwu 
Integration tests are not executed by CI, so this is essentially almost dead code.
The maintainance of the dependencies for this separate module is going to be a pain on the long run.
I believe we have better ways with Github actions to run intgration tests without adding cross-dependencies like this. Working on that part.